### ISSUE TITLE
[plot_wind_rose] Is this used? If not convert.

### ISSUE THREAD

**Issue by stephenholleran** [brightwind-dev/brightwind#52]
Is bw.analyse.plot.plot_wind_rose() used for anything anymore?

If not then rename and convert this to plot a rose for any variable sent. 
`bw.analyse.plot.plot_rose(var_series, direction_series, sectors=12, aggregation_method='frequency' return_data=False)`

var_series can be any time series variable e.g. temperature, shear, pressure, wind speed.
direction_series is the wind direction time series
sectors is the number of direction sectors to split the data into.
aggregation_method is aggregate the data in each sector by count (as frequency), count (as percentage), mean, sum.

I think a lot of this code is used in the freq_table code and possibly in 12x24 too.

Let me know what you think?

**Comment by Inder95** [comment]
> Is bw.analyse.plot.plot_wind_rose() used for anything anymore?
> 
> If not then rename and convert this to plot a rose for any variable sent.
> `bw.analyse.plot.plot_rose(var_series, direction_series, sectors=12, aggregation_method='frequency' return_data=False)`
> 
> var_series can be any time series variable e.g. temperature, shear, pressure, wind speed.
> direction_series is the wind direction time series
> sectors is the number of direction sectors to split the data into.
> aggregation_method is aggregate the data in each sector by count (as frequency), count (as percentage), mean, sum.
> 
> I think a lot of this code is used in the freq_table code and possibly in 12x24 too.
> 
> Let me know what you think?

No it is not used anywhere, plot_wind_rose() is the precursor to plot_wind_rose_with_gradient(). Yes we kept it because of the reason pointed above but there is no good way to use it. 

But as in our library most  of the plots are actually from analyse, hence in keeping with that format I think instead of using
```
bw.analyse.plot.plot_rose(var_series, direction_series, sectors=12, aggregation_method='frequency' return_data=False)
```

we could return it with the function:
```
distribution_by_dir_sector(var_series, direction_series, sectors=12, aggregation_method='%frequency',
                               direction_bin_array=None, direction_bin_labels=None)
```
Yes frequency table can also be used to plot a rose but it is an overkill as we might not need the bins of the other variable. For 12x24 I don't think they share any code, for the data at least.

**Comment by stephenholleran** [comment]
This was included in the #46 branch and #97 merge along with changes to the freq_table().
